packages/mgt-chat/src/statefulClient/useGraphChatClient.ts

```diff
@@ -10,7 +10,9 @@ import { StatefulGraphChatClient } from './StatefulGraphChatClient';
 
 export const useGraphChatClient = (chatId: string): StatefulGraphChatClient => {
   const [chatClient] = useState<StatefulGraphChatClient>(new StatefulGraphChatClient());
-  chatClient.chatId = chatId;
+  if (chatClient.chatId !== chatId) {
```

Comment:
do you need this guard now the that the chatId setter has a check on the existing value?